mRemoteV1/Properties/Settings.settings

```diff
@@ -683,5 +683,8 @@
     <Setting Name="StartUpPanelName" Type="System.String" Scope="User">
       <Value Profile="(Default)">General</Value>
     </Setting>
+    <Setting Name="ConDefaultMultiMonitor" Type="System.Boolean" Scope="User">
```

Comment:
You will need to also add a InDefaultMultiMonitor setting as well, to control the default inheritance.

Reply:
Added, Thanks for review.